pyproject.toml

```diff
@@ -1,2 +1,10 @@
+[build-system]
+requires = [
+    "setuptools >= 40.8.0",
+    "setuptools_scm >= 2.0.0, <4",
```

Comment:
Why do we need `setuptools_scm` here? this project takes its version from https://github.com/pulp/pulp-smash/blob/master/VERSION instead of from scm tags.

Reply:
actually I just copied the example solution from the comment: https://github.com/tox-dev/tox/blob/master/pyproject.toml#L1
I'll remove that